src/components/SearchOfflinePanel.js

```diff
@@ -0,0 +1,14 @@
+import { h } from 'preact'
+import { useI18n } from 'hooks'
+
+export const SearchOfflinePanel = () => {
```

Comment:
Consider renaming things (component, classes, image, message) since this is not specific to search anymore.